br/backup-and-restore-faq.md

```diff
@@ -38,12 +38,18 @@ Running BR with the root access might fail due to the disk permission, because t
 >
 > Therefore, It is recommended to check the permission before data restoration.
 
-## What should I do to resolve the `Io(Os...)` error?
+## How should I handle the `Io(Os...)` error?
 
 Almost all of these problems are system call errors that occur when TiKV writes data to the disk. You can check the mounting method and the file system of the backup directory, and try to back up data to another folder or another hard disk.
  
 For example, you might encounter the `Code: 22(invalid argument)` error when backing up data to the network disk built by `samba`.
 
+## How should I handle the `rpc error: code = Unavailable desc =...` error occurred in BR?
+ 
+This error might occur when the capacity of the cluster to restore (using BR) is insufficient. You can further confirm the cause by checking the monitoring metrics of this cluster or the TiKV log.
+
+To handle this issue, you can try to scale out the cluster resources, reduce the concurrency during restoration, and enable the `RATE_LIMIT` option.
```

Comment:
`restore` is a noun used in our documentation and some other IT companies such as CockroachDB.
```suggestion
To handle this issue, you can try to scale out the cluster resources, reduce the concurrency during restore, and enable the `RATE_LIMIT` option.
```

Reply:
got it:)